nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -691,4 +672,264 @@ private class LogEntryEventHandler_Tests {
+                        LimitsFutureCallsMax__c,
+                        LimitsFutureCallsUsed__c,
+                        LimitsHeapSizeMax__c,
+                        LimitsHeapSizeUsed__c,
+                        LimitsMobilePushApexCallsMax__c,
+                        LimitsMobilePushApexCallsUsed__c,
+                        LimitsQueueableJobsMax__c,
+                        LimitsQueueableJobsUsed__c,
+                        LimitsSoqlQueriesMax__c,
+                        LimitsSoqlQueriesUsed__c,
+                        LimitsSoqlQueryLocatorRowsMax__c,
+                        LimitsSoqlQueryLocatorRowsUsed__c,
+                        LimitsSoqlQueryRowsMax__c,
+                        LimitsSoqlQueryRowsUsed__c,
+                        LimitsSoslSearchesUsed__c,
+                        LimitsSoslSearchesMax__c,
+                        Log__c,
+                        LoggingLevel__c,
+                        LoggingLevelOrdinal__c,
+                        Message__c,
+                        MessageTruncated__c,
+                        Id,
+                        Name,
+                        OriginType__c,
+                        OriginLocation__c,
+                        RecordId__c,
+                        RecordJson__c,
+                        RecordSObjectClassification__c,
+                        RecordSObjectType__c,
+                        RecordSObjectTypeNamespace__c,
+                        StackTrace__c,
+                        Timestamp__c,
+                        TransactionEntryNumber__c,
+                        TriggerIsExecuting__c,
+                        TriggerOperationType__c,
+                        TriggerSObjectType__c
+                    FROM LogEntries__r
+                )
+            FROM Log__c
+            ORDER BY StartTime__c DESC
+            LIMIT 1
+        ];
+    }
+
+    private static LoggerSObjectHandler__mdt getMockConfiguration() {
+        Schema.SObjectType sobjectType = Schema.LogEntryEvent__e.SObjectType;
+        return new LoggerSObjectHandler__mdt(IsEnabled__c = true, SObjectType__c = sobjectType.getDescribe().getName());
+    }
+
+    private static void validateLogFields(LogEntryEvent__e logEntryEvent, Log__c log) {
+        Organization org = [SELECT Id, Name, InstanceName, IsSandbox, NamespacePrefix, OrganizationType, TrialExpirationDate FROM Organization];
+        User currentUser = getCurrentUser();
+        String orgEnvironmentType;
+        if (org.IsSandbox == true && org.TrialExpirationDate != null) {
+            orgEnvironmentType = 'Scratch Org';
+        } else if (org.IsSandbox == true) {
+            orgEnvironmentType = 'Sandbox';
+        } else {
+            orgEnvironmentType = 'Production';
+        }
+
+        System.assertEquals(logEntryEvent.ApiVersion__c, log.ApiVersion__c);
+        System.assertEquals(logEntryEvent.Locale__c, log.Locale__c);
+        //System.assertEquals(logEntryEvent.CreatedById, log.LoggedBy__c);
```

Comment:
remove these commented out fields?

Reply:
I think I meant to solve this in an older PR - the issue is that `logEntryEvent.CreatedById` is null in this context (it's populated in the platform-event trigger), and there's not a way to re-query the platform event like you would a "normal" SObject.

But it turns out, I can just use `UserInfo.getUserId()` instead, so I'll uncomment + fix these 2 fields